elastalert/config.py

```diff
@@ -49,7 +49,7 @@ def get_module(module_name):
     return module
 
 
-def load_configuration(filename):
+def load_configuration(filename, testing=False):
```

Comment:
Please add doc comments explaining what the `testing` flag is and how/when to use it.

In general, I think it's confusing to refer to this as `testing` since it's not unit testing.
Maybe `dry_run` or something?